memgpt/server/ws_api/server.py

```diff
@@ -103,5 +104,14 @@ async def handle_client(self, websocket, path):
 
 
 if __name__ == "__main__":
-    server = WebSocketServer()
+    port = WS_DEFAULT_PORT
```

Comment:
Why not just put this in a function, so you can create the server from other parts of the code?

Reply:
yeah true good point

Reply:
refactored